src/components/StakingHierarchy.js

```diff
@@ -0,0 +1,446 @@
+    }
+  }
+
+  aside::after {
+    border-image: linear-gradient(to bottom, var(--color), var(--next-color)) 1
+      100%;
+    --scale: ${({ number }) => 1.05 + number / 70};
+    --translate: ${({ number }) => number}px;
+    transform: scale(var(--scale)) translateY(var(--translate));
+  }
+`
+
+const Header = styled.div`
+  grid-area: header;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.5rem;
+  h2 {
+    margin: 0;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    align-items: center;
+    h2 {
+      text-align: center;
+    }
+  }
+`
+
+const Pills = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.25rem;
+  p {
+    padding: 0.125rem 0.375rem;
+    white-space: nowrap;
+  }
+`
+
+const Content = styled.div`
+  grid-area: content;
+  margin: 1rem 0 3rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    margin: 0;
+  }
+`
+
+const FlexCentered = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const Glyph = styled(FlexCentered)`
+  grid-area: glyph;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    grid-area: content;
+    svg {
+      width: 50%;
+      height: 50%;
+      opacity: 0.1;
+    }
+  }
+`
+
+const Ether = styled(FlexCentered)`
+  grid-area: ether;
+  z-index: 2;
+  max-width: 5rem;
+  margin: 0 auto;
+`
+
+const StyledEtherSvg = styled(EtherSvg)`
+  --size: ${({ size }) => size};
+  width: var(--size);
+  height: var(--size);
+`
+
+const Line = styled.aside`
+  grid-area: decorator;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  font-size: 28px;
+  position: relative;
+  &::after {
+    content: "";
+    height: 100%;
+    border-left: solid 4px orange;
+    position: absolute;
+    left: calc(50% - 2px);
+    top: 0;
+    z-index: 1;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    display: none;
+  }
+`
+
+const Gold = styled.span`
+  color: ${({ theme }) => theme.colors.stakingGold};
+  font-weight: 600;
+`
+
+const StakingHierarchy = () => {
+  return (
+    <Container>
+      <Section number="1">
+        <Ether>
+          <StyledEtherSvg size="100%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Solo home staking</h2>
+          <Pills className="subtext">
+            <p>
+              <em>Most impactful</em>
+            </p>
+            <p>Full control</p>
+            <p>Full rewards</p>
+            <p>Trustless</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <SoloGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Solo staking on the Ethereum Beacon Chain is the{" "}
+            <Gold>gold standard</Gold> for staking. It is trustless, provides
+            full participation rewards, and improves the decentralization of the
+            network.
+          </p>
+          <p>
+            Those considering solo staking should have at least 32 ETH, hardware
```

Comment:
```suggestion
            Those considering solo staking should have at least 32 ETH, a dedicated computer
```
not sure if "hardware" is a bit intimidating?